CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ BUG FIXES:
 
 * auth/aws: Restrict region selection when in the aws-us-gov partition to avoid IAM errors [[GH-9947](https://github.com/hashicorp/vault/pull/9947)]
 * core: Fix deadlock in handling EGP policies
+* core (enterprise): Fix extraneous error messages in DR Cluster [[GH-1491](https://github.com/hashicorp/vault-enterprise/pull/1491)]
```

Comment:
```suggestion
* core (enterprise): Fix extraneous error messages in DR Cluster
```
We don't link to enterprise PRs - I think it's mostly because not everyone can see them (https://github.com/hashicorp/vault/pull/10056/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR54 for an example.

Reply:
Done!